### PR TITLE
Fix setNftMetadata params & base response

### PR DESCRIPTION
**Pull request by woomurf**
- add collectionId params to setNftMetadata.
  - Users can put values in the desired param. Since ERC721 usually uses contractAddress and Native NFT uses CollectionId, we have modified it to flexibly accommodate both. 
- Fix sendRequest response.
  - When the response data is empty, there is a case where the user does not receive the intended message, so it is modified.

### CODE REVIEW

File: src/nft.ts [hunk 1]
```diff
@@ -147,28 +149,48 @@ export default class Nft extends AinftBase {
...
+  }: SetAinNftMetadataParams): Promise<any>;
   async setNftMetadata({
     appId,
     chain,
     network,
     contractAddress,
     tokenId,
     metadata,
-  }: SetNftMetadataParams): Promise<NftMetadata> {
+  }: SetETHNftMetadataParams): Promise<NftMetadata>;
```
Comment (platfowner) [inline src/nft.ts:167]: What about SetEthNftMetadataParams like SetAinNftMetadataParams?

**Review by platfowner — APPROVED** [review]
LGTM with some comments.
Thanks!

File: src/nft.ts (hunk 1 above)
Comment (woomurf) [reply src/nft.ts]: Done. Thanks!

**Comment by woomurf** [comment]
Thanks for the review!

**Pull request merged**